pages/common/sleep.md

```diff
@@ -14,3 +14,11 @@
 - Delay in hours:
 
 `sleep {{hours}}h`
+
+- Pause for 1 day 3 hours:
+
+`sleep 1d 3h`
+
+- Execute a command after 20 minutes delay:
+
+`sleep 20m && play alarm.mp3`
```

Comment:
```suggestion
`sleep 20m && {{command}}`
```
Or if the command can be a single string we can use this

```suggestion
`sleep 20m && "{{command}}"`
```

NOTE: `{{}}` is an indicator for our clients to highlight placeholders.